scannerlib/fileset/file_set.go

```diff
@@ -36,6 +36,9 @@ import (
 
 const maxTraversalDepth = 100
 
+// Wildcard token value to be used in Scan configurations which will be replaced with the previous check result at runtime
```

Comment:
replace "configurations" with "instructions"
there's a ScanConfig struct that this might be mixed up with otherwise - https://github.com/google/localtoast/blob/master/scannerlib/proto/api.proto#L27

Reply:
done!